errors/errors.go

```diff
@@ -101,169 +98,3 @@ func (e Error) ABCICode() uint32 {
 func (e Error) Codespace() string {
 	return e.codespace
 }
-
-// Is check if given error instance is of a given kind/type. This involves
-// unwrapping given error using the Cause method if available.
-func (e *Error) Is(err error) bool {
-	// Reflect usage is necessary to correctly compare with
-	// a nil implementation of an error.
-	if e == nil {
-		return isNilErr(err)
-	}
-
-	for {
-		if err == e {
-			return true
-		}
-
-		// If this is a collection of errors, this function must return
-		// true if at least one from the group match.
-		if u, ok := err.(unpacker); ok {
-			for _, er := range u.Unpack() {
-				if e.Is(er) {
-					return true
-				}
-			}
-		}
-
-		if c, ok := err.(causer); ok {
-			err = c.Cause()
-		} else {
-			return false
-		}
-	}
-}
-
-// Wrap extends this error with an additional information.
-// It's a handy function to call Wrap with sdk errors.
-func (e *Error) Wrap(desc string) error { return Wrap(e, desc) }
-
-// Wrapf extends this error with an additional information.
-// It's a handy function to call Wrapf with sdk errors.
-func (e *Error) Wrapf(desc string, args ...interface{}) error { return Wrapf(e, desc, args...) }
-
-func isNilErr(err error) bool {
-	// Reflect usage is necessary to correctly compare with
-	// a nil implementation of an error.
-	if err == nil {
-		return true
-	}
-	if reflect.ValueOf(err).Kind() == reflect.Struct {
-		return false
-	}
-	return reflect.ValueOf(err).IsNil()
-}
-
-// Wrap extends given error with an additional information.
-//
-// If the wrapped error does not provide ABCICode method (ie. stdlib errors),
-// it will be labeled as internal error.
-//
-// If err is nil, this returns nil, avoiding the need for an if statement when
-// wrapping a error returned at the end of a function
-func Wrap(err error, description string) error {
```

Comment:
`Wrap` ,`Wrapf` and users were used in external modules and the SDK. It would be nice to keep the public method of the package but with a Go stdlib implementation for a smooth transition. Methods can be deprecated of course.

Reply:
make sense, ill add them